dist/lob-api-bundled.yml

```diff
@@ -21236,10 +21199,7 @@ paths:
             --header 'Content-Type: application/json' \
             -u YOUR_KEY_HERE: \
             --data-raw '{
-                "campaignId": "cmp_f33809b18b6f3ea8",
-                "columnMapping": {
-                    "name": "recipient"
-                }
```

Comment:
Instead of just removing `columnMapping` should it be replaced with `mergeVariableColulmnMapping` or any of the other column mapping properties?

Reply:
Those properties are already mentioned and [available publicly](https://docs.lob.com/#tag/Uploads/operation/uploads_list). So now i'm only removing all mentions of the deprecated properties.
Also, FWIW, this document is generated automatically with `npm bundle`